crates/precompile/bench/blake2.rs

```diff
@@ -87,30 +87,4 @@ pub fn add_benches(group: &mut BenchmarkGroup<'_, criterion::measurement::WallTi
             black_box(blake2::run(black_box(input), u64::MAX).unwrap());
         });
     });
-
-    // Benchmark just the compression function with different round counts
-    group.bench_function("blake2/compress_12_rounds", |b| {
```

Comment:
I think it might make sense to benchmark precompiles and the underlying cryptography separately -- we do still benchmark 12 rounds via the `run` method so we don't lose much here